Frameworks/CoreGraphics/CGImage.mm

```diff
@@ -360,13 +360,13 @@ CGDataProviderRef CGImageGetDataProvider(CGImageRef img) {
 
     const unsigned int stride = CGImageGetBytesPerRow(img);
     const unsigned int size = CGImageGetHeight(img) * stride;
-    woc::unique_iw<unsigned char> buffer(static_cast<unsigned char*>(IwMalloc(size)));
+    std::unique_ptr<unsigned char> buffer(static_cast<unsigned char*>(malloc(size)));
```

Comment:
@bviglietta good point, I should switch to use new to allocate the buffer instead of malloc.